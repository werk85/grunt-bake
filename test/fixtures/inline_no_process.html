<!DOCTYPE html>
<html>
	<head></head>
	<body>
		<!--(bake includes/first.html _process="false" title="Hello World")-->

		<!--(bake includes/first.html _process="true" title="Hello World")-->
	</body>
</html>