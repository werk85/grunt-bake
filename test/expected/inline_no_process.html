<!DOCTYPE html>
<html>
	<head></head>
	<body>
		<div><!--(bake second.html)--></div>

		<div><span>Hello World</span></div>
	</body>
</html>